sphinx_source/json.rst

```diff
@@ -0,0 +1,210 @@
+.. meta::
+    :description: Bu bölümde json sınıfının fonksiyonları anlatılıyor.
```

Comment:
JSON kısaltması "JSON" olarak yazılmalı, "json" ya da "Json" olarak değil. Bu, sadece bu satır için değil, tüm belge için geçerlidir.